lib/config/migration.ts

```diff
@@ -5,7 +5,7 @@ import { logger } from '../logger';
 import { clone } from '../util/clone';
```

Comment:
Question: does this PR fully resolve issue #11459? Or is more work needed after this PR gets merged?

Reply:
Not fully. All migrations should be refactored by the new way https://github.com/renovatebot/renovate/issues/11457. Sorting of refactored migrated properties will be saved.